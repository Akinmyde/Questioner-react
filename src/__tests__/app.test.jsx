import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('Landing Page', () => {
  it('should render app correctly', () => {
    const app = shallow(<App />);
    expect(app).toMatchSnapshot();
  });
});

// import React from 'react';
// import ReactDOM from 'react-dom';
// import App from '../App';

// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });


// "\\.(css|less|sass|scss)$": "<rootDir>/src/setupTests.js",
// "\\.(js|jsx)$": "<rootDir>/src/setupTests.js"
