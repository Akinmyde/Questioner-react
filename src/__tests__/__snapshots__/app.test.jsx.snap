// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page should render app correctly 1`] = `
<Fragment>
  <ToastContainer
    autoClose={5000}
    bodyClassName={null}
    className={null}
    closeButton={
      <CloseButton
        ariaLabel="close"
      />
    }
    closeOnClick={true}
    draggable={true}
    draggablePercent={80}
    hideProgressBar={false}
    newestOnTop={false}
    pauseOnHover={true}
    pauseOnVisibilityChange={true}
    position="top-right"
    progressClassName={null}
    rtl={false}
    style={null}
    toastClassName={null}
    transition={[Function]}
  />
  <div>
    <Header />
    <Switch>
      <Route
        component={[Function]}
        path="/login"
      />
      <Route
        component={[Function]}
        path="/signup"
      />
      <Route
        component="not-found"
        path="/not-found"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Redirect
        to="/not-found"
      />
    </Switch>
    <Footer />
  </div>
</Fragment>
`;
